Week2/database.js

```diff
@@ -0,0 +1,58 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world'
+});
+
+
+
+//Part 1 - 1
+let country_name = 'Syria';
+let capitalOfCountry = 'select country.name, city.name from city inner join country on city.id = country.capital where country.name = ' + connection.escape(country_name);
+
+//Part 1 - 2
+let country_region = 'Western Europe';
+let findingSpokenLanguages = 'select countryLanguage.language, country.region from country join countryLanguage on country.Code = countryLanguage.countryCode where country.region =' + connection.escape(country_region);
+
+//Part 1 - 3
+let country_language = 'Dutch';
+let findingNumberOfCities = 'select count(*) as numberOfCities from city join countryLanguage on city.countryCode = countryLanguage.countryCode where language = ' + connection.escape(country_language);
+
+//Part 1 - 4
+let display_countries = "select name, region from country as a join countryLanguage as b on a.code = b.countryCode where b.Language = 'French' and b.IsOfficial = 'T' and region in (select region from country as a join countryLanguage as b on a.code = b.countryCode where b.Language = 'French' and b.IsOfficial = 'T' group by region having COUNT(*) > 1 )"
```

Comment:
Are there any countries that have A) Same official *language* B) Same *region* If yes, display those countries. If no, display TRUE or FALSE

You could take both language and region as inputs as such:

```
function display_countries(language, region) {
    return `select name, region
            from country as a 
            join countryLanguage as b on a.code = b.countryCode 
            where b.Language = ${connection.escape(language)} and b.IsOfficial = 'T' and region = ${connection.escape(region)}
    `;
  }
```